exotica_core/src/problems/dynamic_time_indexed_shooting_problem.cpp

```diff
@@ -299,6 +354,11 @@ Eigen::VectorXd DynamicTimeIndexedShootingProblem::Simulate(Eigen::VectorXdRefCo
     return scene_->GetDynamicsSolver()->Simulate(x, u, tau_);
 }
 
+Eigen::VectorXd DynamicTimeIndexedShootingProblem::GetControlLimits() const
+{
+    return parameters_.ControlLimits;
```

Comment:
Do they get checked for size and validity on initialisation? I.e., should all be positive and vector size should be correct. If vector size == 1, set same limit for all joints.